ArdusatSDK.cpp

```diff
@@ -364,8 +369,8 @@ int calculateCheckSum(const char *sensor_name, int num_vals, ...) {
 /*
  * toCSV Output functions
  */
-void _output_buffer_reset() {
-  memset(_output_buffer, 0, 512);
+void _resetOutBuf() {
+  memset(_getOutBuf(), 0, OUTPUT_BUF_SIZE);
```

Comment:
To try and save some space when the sd card logging libraries were included, one of the buffers there was cleverly repurposed by ben. But without the logging library a new buffer is needed. However, if the logging library is included in addition to this one, it would take too much space to allocate a new buffer on top of the originally repurposed one necessary for the sd card... if that makes sense. So the `_getOutBuf()` function is necessary for dynamically allocating a new buffer, or repurposing the one in the sd card library.